docs/Documentation/Events-List.md

```diff
@@ -250,6 +250,16 @@ Works the same way as a normal tag event, but instead of setting a tag for one p
     globaltag add global_areNPCsAgressive
     ```
 
+## Hunger: `hunger`
+
+This event changes the food level of the player. The second argument is the modification type. There are `give`/`add`, `take` and `set`. The second argument is the amount. With `set` can the food level be anything. If `give` or `take` is specified and the final amount won't be more than 20 or less than 0. If the hunger level is below 7, the player cannot sprint.
```

Comment:
Dont introduce aliases like `give/take`

Reply:
Do you mean aliases like give and add? Or do you want me to remove take because the same result could be achieved with `give -5` for example?

Reply:
i mean aliases like `give`/`add`, its the same. Sorry for the confusion

Reply:
okay no worries, that's what I thought. I just wanted to confirm before pushing anything